docs/web3-eth-abi.rst

```diff
@@ -206,7 +206,8 @@ Example
     > "0x000000000000000000000000000000000000000000000000000000008bd02b7b0000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000748656c6c6f212500000000000000000000000000000000000000000000000000"
 
     web3.eth.abi.encodeParameters(['uint8[]','bytes32'], [['34','434'], '0x324567fff']);
```

Comment:
Thanks for opening this PR. This would be the correct example:

 ``` javascript 
web3.eth.abi.encodeParameters(['uint8[]','bytes32'], [['34','43'], '0x0000000000000000000000000000000000000000000000000000000324567fff'])
> 0x00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000324567fff00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000022000000000000000000000000000000000000000000000000000000000000002b
```

Reply:
Sorry yes! fixed it.